src/devices/MettlerToledo/README.md

```diff
@@ -0,0 +1,40 @@
+﻿# Mettler Toledo
+
+## Summary
+This binding provides a higher-level interface for interacting with [Mettler Toledo](https://www.mt.com)-brand scales.
```

Comment:
missing enter before to have a proper markdown. I can recommend to use a markdown linter. You haver quite some choice. Most fix automatically those kinds of nit.